src/kaleidoscope/hardware/ATMegaKeyboard.h

```diff
@@ -67,44 +67,182 @@ struct cRGB {
   uint8_t BOARD::debounce_matrix_[matrix_rows][matrix_columns]; \
```

Comment:
Should 'BOARD' here change to KeyboardHardware?

Reply:
No, because we need the class name here, not the instance name.